packages/components/Menu/src/MenuItem/useMenuItem.ts

```diff
@@ -104,7 +104,11 @@ export const useHoverFocusEffect = (hovered: boolean, componentRef: React.Mutabl
     if (hovered) {
       componentRef?.current?.focus();
     } else {
-      componentRef?.current?.blur();
+      // Since MenuList uses FocusZone on macOS, we need to handle the .blur() logic there in order to
```

Comment:
for my own understanding? why does blur logic interfere with keyboard navigation?

Reply:
When we blur on mouseExit, focus is lost from the menu overall. Then, the user can't ArrowUp/Down back between menu items until they put their mouse back over the menu.

Reply:
can we make sure we capture that in comment and the pr description please?